app/controllers/concerns/saml_idp_logout_concern.rb

```diff
@@ -1,6 +1,8 @@
-# rubocop:disable Metrics/ModuleLength
+# rubocop:disable ModuleLength
+# TODO(sbc): Refactor to address rubocop warning
```

Comment:
I don't think we care about the module length here because everything in this module is related to single logout. I don't think this module is doing too much and it's probably not worth our time to make it fit in 100 lines.